js/settings.json

```diff
@@ -4,10 +4,13 @@
     "export"
   ],
   "tools": [
-    "pencil"
+    "pencil",
+    "oscCircles",
+    "heart"
```

Comment:
it looks like u're trying to load a "heart" tool, but there isn't any heart tool in ur repo (maybe it's part of another branch)? this prevents the app from loading. so u'll either need to include the heart tool file in this PR or remove this line from the settings files

Reply:
@shmoopyyy u know what, never mind, this is such a small change i'll just resolve it as i merge (but keep this in mind in the future though)